src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -529,6 +530,10 @@ def create_export(self, fs_name, cluster_id, pseudo_path, read_only, path):
             if not self.check_fs(fs_name):
                 return -errno.ENOENT, "", f"filesystem {fs_name} not found"
 
+            if not isabs(pseudo_path) or pseudo_path == "/":
```

Comment:
Consider someone passing "//" as a pseudo_path. We need to collapse the path. Python has a method for this `os.path.normpath`.

Reply:
> Consider someone passing "//" as a pseudo_path. We need to collapse the path. Python has a method for this `os.path.normpath`.

Normpath doesn't work for leading double slash:
https://stackoverflow.com/questions/7816818/why-doesnt-os-normpath-collapse-a-leading-double-slash